app/src/main/java/com/example/androidlearning/MainActivity.kt

```diff
@@ -1,11 +1,54 @@
 package com.example.androidlearning
 
+import android.content.Intent
 import androidx.appcompat.app.AppCompatActivity
 import android.os.Bundle
+import android.util.Log
+import android.widget.Button
 
 class MainActivity : AppCompatActivity() {
+    private lateinit var goToBtn: Button
+
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_main)
+        Log.i("Abhi","Activity1: onCreate")
+
+        goToBtn = findViewById(R.id.goTobtn)
```

Comment:
This is old concept how we get views, check viewbinding you will feel its easier.

Reply:
done, yes its very easy